docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  app:
+    build:
+      context: ./src/GeositeFramework
+    working_dir: /usr/src/
+    ports:
+      - "54633:54633"
```

Comment:
Can we have this run on port 54634? That'd make it possible to serve both the static site and the existing site simultaneously.